### ISSUE TITLE
GraphQL learning resources

### ISSUE THREAD

**Issue by jackcarlisle** [dwyl/learn-graphQL#3]

**Comment by jackcarlisle** [comment]
Great talk given by Nick Schrock - https://www.youtube.com/watch?v=Ed6oJXKt3-M
Relay visual learners - https://sgwilym.github.io/relay-visual-learners/

**Comment by nelsonic** [comment]
@jackcarlisle did you see: https://medium.com/@clayallsopp/your-first-graphql-server-3c766ab4f0a2 ?

**Comment by jackcarlisle** [comment]
@nelsonic no but it looks good. I'll check it it out!

**Comment by nelsonic** [comment]
- Videos: http://graphql.org/docs/videos/
- Type System: http://graphql.org/docs/api-reference-type-system/
- Introduction To GRAPHQL by Lee Byron - https://www.youtube.com/watch?v=Wq02BNrN1dU
- Build a GraphQL server for Node.js, using PostgreSQL/MySQL - https://youtu.be/DNPVqK_woRQ & https://github.com/leebenson/graphql-with-sequelize
- How to run your own GraphQL: http://pcarion.com/2015/09/22/embed-graphql-server/
- https://github.com/chentsulin/awesome-graphql
- [http://graphql-swapi.parseapp.com/](http://graphql-swapi.parseapp.com/?query=query%20%7B%0A%20%20allFilms%20%7B%0A%09%09films%20%7B%0A%20%20%20%20%20%20title%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D) (_until Facebook shuts down Parse  in January 2017..._) >> code: https://github.com/graphql/swapi-graphql/
- https://learngraphql.com/basics/introduction (_shared by_ @nikhilaravi )

**Comment by jackcarlisle** [comment]
http://pcarion.com/2015/09/26/graphql-resolve/
Lee Byron video - https://www.youtube.com/watch?v=cr4QB3j8qFc

**Comment by Jbarget** [comment]
GraphQL + Phoenix: https://ryanswapp.com/2016/11/29/phoenix-graphql-tutorial-with-absinthe/

**Comment by nelsonic** [comment]
@Jbarget yeah, followed it: https://github.com/nelsonic/phoenix-absinthe-graphql-api ✅ 
and our App's API will have GraphQL: https://github.com/dwyl/api/blob/5c95b4479da3f5499ba73f7967cc2e2ca0c1b562/lib/api/web/router.ex#L16-L20 😉 
Just need to get through my "DevOps" (_client_) work so I can get back to the API...! 😖

**Comment by newswim** [comment]
This one might be obvious but didn't see it listed:
- https://www.howtographql.com
